pkg/asset/installconfig/gcp/validation.go

```diff
@@ -133,3 +133,18 @@ func validateMachineNetworksContainIP(fldPath *field.Path, networks []types.Mach
 	}
 	return field.ErrorList{field.Invalid(fldPath, subnetName, fmt.Sprintf("subnet CIDR range start %s is outside of the specified machine networks", ip))}
 }
+
+func CheckForServicesToBeEnabled(projectServices []string) error {
+	services := map[string]string{"compute.googleapis.com": "enabled", "cloudapis.googleapis.com": "enabled",
```

Comment:
try using https://godoc.org/k8s.io/apimachinery/pkg/util/sets

that package provides an easy `sets.NewString(<list of string values>)` to easily run a check `Has(<service>)`